src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +57,42 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        if (dateParams.length == 0) {
+            return Optional.empty();
+        }
+        return getOptional("" + dateParams[0] + dateParams[1] + dateParams[2],FORMATTER1);
```

Comment:
```suggestion
        return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
```
 + use try catch here

Reply:
But why I need use here try catch if I had it in my custom method?

Reply:
yeah, that can work as well